src/app/contexts/EventTrackingContext/index.tsx

```diff
@@ -58,7 +59,7 @@ const getCampaignID = (pageType: CampaignPageTypes) => {
     [ARTICLE_PAGE]: 'article',
     [MEDIA_ARTICLE_PAGE]: 'article-sfv',
     [FRONT_PAGE]: 'index-home',
-    [MEDIA_PAGE]: 'player-episode-tv',
+    [MEDIA_PAGE]: 'player-episode',
```

Comment:
was anything actually meant to be `player-episode-tv`?

Reply:
See my comment here: https://paper.dropbox.com/doc/Test-Plan-On-Demand-Audio-TV-and-Live-Radio-pages-anBxRcoM9CiAJNdnkB6I1?t=948328379551413787286778#:t=948328379551413787286778

That said... might it need to be `player-episode-tv` for ATI when viewing a shared media component such as recent episodes or the radio schedule. Do we need to check with Product before we remove this?

Reply:
I guess lug it in with `player-episode-tv`, as long as they're aware it's changing it should be fine